cast/js/rhino/src/main/java/com/ibm/wala/cast/js/translator/RhinoToAstTranslator.java

```diff
@@ -1596,6 +1600,40 @@ public CAstNode visitSymbol(Symbol node, WalkContext arg) {
       return null;
     }
 
+    @Override
+    public CAstNode visitTemplateCharacters(TemplateCharacters node, WalkContext arg) {
+      return Ast.makeConstant(node.getValue());
+    }
+
+    @Override
+    public CAstNode visitTemplateLiteral(TemplateLiteral node, WalkContext arg) {
+      List<AstNode> elements = node.getElements();
+      if (elements.size() == 1) {
+        return this.visit(elements.get(0), arg);
+      }
+      CAstNode lastBinaryNode =
+          Ast.makeNode(
+              CAstNode.BINARY_EXPR,
+              translateOpcode(Token.ADD),
+              visit(elements.get(elements.size() - 2), arg),
+              visit(elements.get(elements.size() - 1), arg));
+      for (int i = elements.size() - 3; i >= 0; i--) {
+        lastBinaryNode =
+            Ast.makeNode(
+                CAstNode.BINARY_EXPR,
+                translateOpcode(Token.ADD),
+                visit(elements.get(i), arg),
+                lastBinaryNode);
+      }
+      return lastBinaryNode;
+    }
+
+    @Override
+    public CAstNode visitTaggedTemplateLiteral(TaggedTemplateLiteral node, WalkContext arg) {
+      // TODO Auto-generated method stub
+      return null;
+    }
```

Comment:
I'm concerned this will cause a crash if we get an input with a tagged template literal.  If we don't handle this case yet, maybe we should not add this method and just skip visiting the nodes?